frontend/src/pages/InducteeSignUpPage/components/SignUpForm/index.js

```diff
@@ -0,0 +1,250 @@
+
+            <Grid item>
+              <Grid container direction='row' spacing={2}>
+                <Grid item xs={8}>
+                  <MajorDropdownField name='major' label='Major' fullWidth />
+                </Grid>
+
+                <Grid item xs={4}>
+                  <YearDropdownField
+                    name='gradYear'
+                    label='Grad Year'
+                    fullWidth
+                  />
+                </Grid>
+              </Grid>
+            </Grid>
+
+            <Grid item>
+              <h3>Review your coursework</h3>
+              <Typography>Description.</Typography>
+            </Grid>
+            <Grid item>
+              <img
+                className={classes.logo}
+                src={FOUR_YEAR_PLAN}
+                alt='Coursework Requirements'
+              />
+            </Grid>
+            <Grid item>
+              <Grid container direction='row' alignItems='center'>
+                <Grid item xs={1}>
+                  <Field
+                    type='checkbox'
+                    component={Checkbox}
+                    name='courseRequirement'
+                    color='primary'
+                  />
+                </Grid>
+                <Grid item>
+                  <Typography>I understand</Typography>
+                </Grid>
+              </Grid>
+            </Grid>
+
+            <Grid item>
+              <h3>Select an info session</h3>
+            </Grid>
+
+            <Grid item>
+              <InfoSessionDropdownField
+                name='infoSession'
+                label='Info Session'
+                fullWidth
+              />
+            </Grid>
+
+            <Grid item>
+              <h3>Buddy related (TBD)</h3>
+            </Grid>
+
+            <Grid item>
+              <h3>Description</h3>
+            </Grid>
+            <Grid item>
+              <Grid container direction='row' alignItems='center'>
+                <Grid item xs={1}>
+                  <Field
+                    type='checkbox'
+                    component={Checkbox}
+                    name='newsletter'
+                    color='primary'
+                  />
+                </Grid>
+                <Grid item>
+                  <Typography>
+                    I understand that I will subscribe to newsletter.
+                  </Typography>
+                </Grid>
+              </Grid>
+            </Grid>
+
+            <Grid item>
+              <Button
+                className={classes.signUp}
+                variant='contained'
+                color='primary'
+                fullWidth
+                disabled={isSubmitting}
+                onClick={submitForm}
+              >
+                Sign Up
+              </Button>
+            </Grid>
+
+            <Grid item>
+              <Grid container justify='center'>
+                <Button
+                  className={classes.signInRedirect}
+                  to={ROUTES.SIGN_IN}
```

Comment:
Use `onClick` prop and `useHistory` hook to navigate to next page on click, remove the `to` and `component` prop